src/main/java/org/wso2/lsp4intellij/utils/DocumentUtils.java

```diff
@@ -38,6 +39,10 @@
     public static final String WIN_SEPARATOR = "\r\n";
     public static final String LINUX_SEPARATOR = "\n";
 
+    public static int getTabSize(Editor editor) {
```

Comment:
```suggestion
    private static int getTabSize(Editor editor) {
```

Reply:
it could be used in EditorEventManager for `FormattingOptions( .. )`, too